src/cudnn/batchnorm.jl

```diff
@@ -68,57 +84,82 @@ function cudnnBNForward!(y::DenseCuArray{T}, g::DenseCuArray{T}, b::DenseCuArray
   return y
 end
 
+function ∇batchnorm(g::Nothing, b::Nothing, x::DenseCuArray, dy::DenseCuArray,
+                  running_mean, running_var, momentum; kws...)
+  g = fill!(similar(x, size(ndims(x)-1)), 1)
+  b = fill!(similar(x, size(ndims(x)-1)), 0)
```

Comment:
```suggestion
  affine_sz = _wsize(x)
  g = fill!(similar(x, affine_sz), 1)
  b = fill!(similar(x, affine_sz), 0)
```
Same as the forward pass.